docs/guides/tooling/typescript-support.mdx

```diff
@@ -71,6 +71,28 @@ If that does not work, try restarting the IDE.
 
 :::
 
+### Processing your Cypress configuration and plugins
+
+Cypress needs to be able to transpile your Cypress configuration and plugins written in TypeScript in order to make them executable within Cypress' Node.js runtime.
+To do this, Cypress will attempt to read the user's TypeScript and project configuration to apply the correct TypeScript loader to Cypress' Node.js runtime.
```

Comment:
Maybe we should point to where it reads this from... ie: /root/tsconfig.json etc